io/slice_long_to_bytes.ts

```diff
@@ -10,7 +10,7 @@
  */
 export function sliceLongToBytes(
   d: number,
-  dest = Array.from<number>({ length: 8 }),
+  dest: number[] = Array.from<number>({ length: 8 }),
```

Comment:
Ditto. Is this needed?

Reply:
See here how `dest` is unsupported. It's a bug in `deno doc --lint`:

https://deno.land/std@0.210.0/io/slice_long_to_bytes.ts?s=sliceLongToBytes